webapp/ember-cli-build.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(defaults) {
           'handlebars.js': null
       },
       'ember-cli-babel': {
-          includePolyfill: not_testing,
```

Comment:
I was following advice from another thread due to issues I stumbled upon: https://slack-redir.net/link?url=https%3A%2F%2Fgithub.com%2Fsamselikoff%2Fember-cli-mirage%2Fissues%2F1084. Before I did that, I had lots of weirder errors